publication-rest/src/main/java/no/unit/nva/publication/fetch/FetchPublicationContextHandler.java

```diff
@@ -0,0 +1,49 @@
+package no.unit.nva.publication.fetch;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.google.common.net.MediaType;
+import no.unit.nva.model.Publication;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.HttpURLConnection;
+import java.util.List;
+import static com.google.common.net.MediaType.JSON_UTF_8;
+import static nva.commons.apigateway.MediaTypes.APPLICATION_JSON_LD;
+
+public class FetchPublicationContextHandler extends ApiGatewayHandler<Void, String> {
+
+    private static final String ENV_API_HOST = "API_HOST";
+    private static final String ENV_CUSTOM_DOMAIN_BASE_PATH = "CUSTOM_DOMAIN_BASE_PATH";
+
+    public FetchPublicationContextHandler() {
+        super(Void.class, new Environment());
+    }
+
+    @Override
+    protected String processInput(Void input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        var host = environment.readEnv(ENV_API_HOST);
+        var path = environment.readEnv(ENV_CUSTOM_DOMAIN_BASE_PATH);
```

Comment:
Use 

```
   import static no.unit.nva.publication.PublicationServiceConfig.ENVIRONMENT;

   …
    public static final String HOST = ENVIRONMENT.readEnv("API_HOST");
    public static final String PATH = ENVIRONMENT.readEnv("CUSTOM_DOMAIN_BASE_PATH");
    …
    var baseUri = UriWrapper.fromHost(HOST)
        .addChild(PATH)
        .getUri();
````

Reply:
Shouldn't the constants be private? See new commit